core/server/api/authentication.js

```diff
@@ -216,8 +216,10 @@ authentication = {
         }
 
         function sendResetNotification(data) {
-            var baseUrl = config.get('forceAdminSSL') ? (config.get('urlSSL') || config.get('url')) : config.get('url'),
-                resetUrl = globalUtils.url.urlJoin(baseUrl, 'ghost/reset', globalUtils.encodeBase64URLsafe(data.resetToken), '/');
+            var baseUrl = config.get('forceAdminSSL') ? globalUtils.url.urlFor('home', {secure: true}, true) : globalUtils.url.urlFor('home', true),
```

Comment:
In this case we are generating the home url/base url, which is the blog url, not the admin url.
I wonder, why do we ask for `config.get('forceAdminSSL')`?I know you didn't change this logic.